w08h03/test/pgdp/datastructures/UnitTest.java

```diff
@@ -39,7 +39,8 @@ <Type extends Comparable<Type>> void testArray(Type[] input) {
 	@Test
 	@DisplayName("should iterate over empty graph")
 	public void emptyTest() {
-		assertThrows(NoSuchElementException.class, () -> (new QuarternarySearchTree<Integer>()).iterator().next());
+		Exception e = assertThrows(NoSuchElementException.class, () -> (new QuarternarySearchTree<Integer>()).iterator().next());
+		assertEquals(e.getMessage(), "Trying to call next on an empty QuarternarySearchTreeIterator");
```

Comment:
You accidentally added the commits for W08H03 😅 If you need help in removing this from the PR, please let me know 👍